src/logo.tsx

```diff
@@ -0,0 +1,352 @@
+import * as React from 'react';
+import {useTheme} from './hooks';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import {SkinName} from './skins/types';
+import {TrackingEvent} from './utils/types';
+import {ExclusifyUnion} from './utils/utility-types';
+import Touchable from './touchable';
+
+type LogoType = 'isotipo' | 'imagotype' | 'vertical';
+type LogoImageProps = {size: number; logoType: LogoType};
```

Comment:
I haven't added a default `size` nor a min value here. Since they're not exported I didn't see it needed, but you'll tell me